clusters/app.ci/prow/03_deployment/private_deck.yaml

```diff
@@ -0,0 +1,296 @@
+                  mountPath: /static/extensions
+                  readOnly: true
+                - name: oauth-config
+                  mountPath: /etc/github
+                  readOnly: true
+                - name: oauth
+                  mountPath: /etc/oauth
+                  readOnly: true
+                - name: job-config-misc
+                  mountPath: /etc/job-config/misc
+                  readOnly: true
+                - name: job-config-master-periodics
+                  mountPath: /etc/job-config/master-periodics
+                  readOnly: true
+                - name: job-config-master-postsubmits
+                  mountPath: /etc/job-config/master-postsubmits
+                  readOnly: true
+                - name: job-config-master-presubmits
+                  mountPath: /etc/job-config/master-presubmits
+                  readOnly: true
+                - name: job-config-3x
+                  mountPath: /etc/job-config/3.x
+                  readOnly: true
+                - name: job-config-41
+                  mountPath: /etc/job-config/4.1
+                  readOnly: true
+                - name: job-config-42
+                  mountPath: /etc/job-config/4.2
+                  readOnly: true
+                - name: job-config-43
+                  mountPath: /etc/job-config/4.3
+                  readOnly: true
+                - name: job-config-44
+                  mountPath: /etc/job-config/4.4
+                  readOnly: true
+                - name: job-config-45
+                  mountPath: /etc/job-config/4.5
+                  readOnly: true
+                - name: job-config-46
+                  mountPath: /etc/job-config/4.6
+                  readOnly: true
+                - name: job-config-47
+                  mountPath: /etc/job-config/4.7
+                  readOnly: true
+                - name: job-config-48
+                  mountPath: /etc/job-config/4.8
+                  readOnly: true
+                - name: job-config-49
+                  mountPath: /etc/job-config/4.9
+                  readOnly: true
+                - name: job-config-410
+                  mountPath: /etc/job-config/4.10
+                  readOnly: true
+                - mountPath: /etc/build-farm-credentials
+                  name: build-farm-credentials
+                  readOnly: true
+              resources:
+                requests:
+                  memory: "3Gi"
+                  cpu: "600m"
+          volumes:
+            - name: service-account-token
+              projected:
+                sources:
+                  - serviceAccountToken:
+                      path: token
+            - name: config
+              configMap:
+                name: config
+            - name: extensions
+              configMap:
+                name: private-deck-poc-extensions
+            - name: oauth
+              secret:
+                secretName: github-credentials-openshift-ci-robot
```

Comment:
Can we make this use the GH App auth instead of the PAT? I'd like to avoid adding new things that still use PATs.

Reply:
I am working on this, but running into issues with the `/pr-status` endpoint. I am still researching what is going wrong, but when I switch to using the app it no longer works.